components/retro-go/rg_audio.c

```diff
@@ -379,6 +406,9 @@ void rg_audio_set_mute(bool mute)
         rg_i2c_gpio_set_direction(AW_HEADPHONE_EN, 0);
         rg_i2c_gpio_set_level(AW_HEADPHONE_EN, !mute);
     #endif
+#elif RG_AUDIO_USE_BUZZER_PIN
```

Comment:
I think this should be an `#if`, it's not inconceivable that a device might have support for both the DAC and the buzzer.